cluster/mesos/docker/docker-compose.yml

```diff
@@ -147,6 +148,7 @@ scheduler:
     --cluster-domain=cluster.local
     --mesos-executor-cpus=1.0
     --v=4
+    --profiling=true
```

Comment:
side effect is that the debugging handler API's are enabled for the kubelet
(this allows, for example, /debug/pprof/... to function)

On Fri, Oct 16, 2015 at 4:38 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In cluster/mesos/docker/docker-compose.yml
> <https://github.com/kubernetes/kubernetes/pull/15735#discussion_r42220142>
> :
>
> > @@ -147,6 +148,7 @@ scheduler:
> >      --cluster-domain=cluster.local
> >      --mesos-executor-cpus=1.0
> >      --v=4
> > +    --profiling=true
>
> Are there any side effects?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15735/files#r42220142>.
>